source/_components/switch.fritzbox_wifi.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "AVM FRITZBOX (Guest) WIFI Switch"
+description: "Instructions on how to integrate your AVM FRITZBOX (Guest) WIFI as switches into Home Assistant."
+date: 2018-12-23 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: avm.png
+ha_category: Switch
+ha_iot_class: "Local Polling"
+---
```

Comment:
Missing: `ha_release:`

Reply:
what should I fill out there if its not released?

Reply:
You can leave it blank until you know in which release it will be merged. Or fill in 0.85.0 if it is likely that the PR of the component will be quickly merged before the next release in Januari.